h2/src/test/org/h2/test/store/TestMVStore.java

```diff
@@ -2075,6 +2072,44 @@ private void testLargerThan2G() {
         FileUtils.delete(fileName);
     }
 
+    private void testNotSerializable() {
+        class Logic {
+            public void run() {
+                String fileName = getBaseDir() + "/" + getTestName();
```

Comment:
You need to delete a file with this name here too (in case it was not deleted after previous test for a some reason).